packages/core/content-manager/server/services/field-sizes.js

```diff
@@ -52,6 +52,10 @@ const createFieldSizesService = ({ strapi }) => {
       return fieldSizes;
     },
 
+    hasFieldSize(type) {
+      return !!fieldSizes[type];
+    },
```

Comment:
Do you think hasFieldSize could be implemented as part of getFieldSize? They seem to always be called together

Reply:
No I think throwing the error is good, so both options you mention are okay. Let's keep it like that then 👍